changelog.md

```diff
@@ -0,0 +1,10 @@
+# Change Log
+This file contains all the notable changes done to the Ballerina Regex package through the releases.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [0.6.3] - 2021-01-22
```

Comment:
I think pre-release tag is missing.

Reply:
It doesn't have a pre-release tag as we don't do any changes after adding APIs in the release version [0.6.0](https://github.com/ballerina-platform/module-ballerina-regex/tree/release-0.6.0).